content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,30 +5,155 @@ weight = 20
 
 +++
 
-## What is Kubeflow?
+This pages describes how to install Kubeflow standalone components or Kubeflow Platform using package
+distributions or raw manifests. Check [the introduction guide](/docs/started/introduction) to
+understand what are Kubeflow standalone components and what is Kubeflow Platform.
 
-Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
-Operators can choose what is best for their users, there is no requirement to deploy every component.
+## Installing Kubeflow
 
-Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
-[Architecture](/docs/started/architecture/) pages.
+You can install Kubeflow components using one of these methods:
 
-## How to install Kubeflow?
+- [**Install Kubeflow Components Standalone**](#install-kubeflow-components-standalone)
+- [**Install Kubeflow Platform from Packaged Distributions**](#install-kubeflow-platform-from-packaged-distributions)
+- [**Install Kubeflow Platform from Raw Manifests**](#install-kubeflow-platform-from-raw-manifests) <sup>(advanced users)</sup>
 
-Anywhere you are running Kubernetes, you should be able to run Kubeflow.
-There are two primary ways to install Kubeflow:
+### Install Kubeflow Components Standalone
 
-1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
-1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
+Kubeflow components can be deployed as standalone applications. You can integrate those components
+to your existing AI/ML platform. This is the easiest method to get started with Kubeflow ecosystem
+since those components usually don't require additional management tools used in Kubeflow Platform.
 
-<a id="packaged-distributions"></a>
-<a id="install-a-packaged-kubeflow-distribution"></a>
+This table points to the installation guide for every Kubeflow component, the GitHub repository,
+and corresponding [stage of ML lifecycle](/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle)
+for every Kubeflow component.
 
-## Packaged Distributions of Kubeflow
+<div class="table-responsive distributions-table">
```

Comment:
What is the user value to order them by popularity and how are we going to track the components popularity in the future ?
E.g. from my point of view, ML Lifecycle makes more sense since order will always be the same, and we can link this table with Kubeflow ML Lifecycle: https://github.com/kubeflow/website/pull/3728.
If we don't like that, we can order them alphabetically. 
@thesuperzapper @kubeflow/kubeflow-steering-committee @juliusvonkohout Thoughts ?

Reply:
Lifecycle or alphabetically is low maintenance

Reply:
Agree on alphabetical ordering - politically correct and low maintenance.